virttest/qemu_monitor.py

```diff
@@ -2536,6 +2567,45 @@ def blockdev_del(self, node_name):
         args = {"node-name": node_name}
         return self.cmd(cmd, args)
 
+    def blockdev_backup(self, device, target, sync, job_id=None, **kwargs):
+        """
+        Backup block device via QMP command blockdev-backup
+
+        :param string device:  device node-name
+        :param string target: target device node-name
+        :param string sync: sync mode, vaules are 'top', 'full', 'none', 'incremental'
+        :param dict kwargs: dict for other options, eg. 'comparess', 'on-source-error'
+        :return: block job ID
+        :rtype: string
+        """
+        cmd = "blockdev-backup"
+        self.verify_supported_cmd(cmd)
+        args = copy.copy(kwargs)
```

Comment:
Need to call `self._build_args` to update `kwargs`.

Reply:
Ok,  will update in next version.